syntax = "proto3";

message ProtoTask {
  string db_name = 1;
  repeated string nlq_tokens = 2;
}

message ProtoCandidates {
  repeated string cq = 1;
}
